### PR TITLE
add new version 5.1.1

### PR DESCRIPTION
**Pull request by riuvshin**
add bugfix version

### CODE REVIEW

File: dockerfiles/cli/version/latest.ver [hunk 1]
```diff
@@ -1 +1 @@
-5.1.0
\ No newline at end of file
+5.1.1
```
Comment (benoitf) [inline dockerfiles/cli/version/latest.ver:1]: will it fail if we have a newline there ?

**Review by benoitf — APPROVED** [review]

File: dockerfiles/cli/version/latest.ver (hunk 1 above)
Comment (riuvshin) [reply dockerfiles/cli/version/latest.ver:1]: idk we never had new line here... I think it will work but not sure

Comment (benoitf) [reply dockerfiles/cli/version/latest.ver:1]: ok up to you if you're a tester or not ^^

Comment (riuvshin) [reply dockerfiles/cli/version/latest.ver:1]: I don;t want break things now, I will test this later if it works I will add new line

**Pull request merged**